mountpoint-s3/src/main.rs

```diff
@@ -342,6 +342,14 @@ fn mount(args: CliArgs) -> anyhow::Result<FuseSession> {
         .transpose()
         .context("Failed to parse endpoint URL")?;
 
+    match retrieve_instance_type() {
+        Ok(ec2_instance_type) => {
+            // TODO: config throughtput_target_gbps here.
+            tracing::debug!("The ec2 instance type is: {:?}", ec2_instance_type);
+        }
+        Err(e) => tracing::error!("failed to retrieve ec2 instance type. {:?}", e),
```

Comment:
Let's make sure AWS service names like EC2 are capitalized correctly where possible - especially in customer-facing bits like logging.